en/backup-to-aws-s3-by-snapshot.md

```diff
@@ -0,0 +1,161 @@
+
+This document describes how to back up a TiDB cluster deployed on AWS Elastic Block Store (EBS) in Kubernetes to AWS S3.
+
+The backup method described in this document is based on the CustomResourceDefinition (CRD) of TiDB Operator. If your TiDB cluster is deployed on AWS EKS and uses EBS volumes, you can use the method described in this document to back up the TiDB cluster.
+
+## Recommended scenarios and limitations
+
+### Recommended scenarios
+
+If you have the following requirements when backing up TiDB cluster data, you can use TiDB Operator to back up the data in volume snapshots and metadata to AWS S3:
+
+- Minimize the impact of backup, for example, to keep the impact on QPS and transaction latency less than 5%, and occupy no cluster CPU and memory.
+- Back up and restore data in a short time. For example, finish backup within 1 hour and restore in 2 hours.
+
+If you have any other requirements, select an appropriate backup method based on [Backup and Restore Overview](backup-restore-overview.md).
+
+### Limitations
+
+- Snapshot backup is only applicable to TiDB Operator v1.4.0 and later, and TiDB v6.3.0 and later.
+- To use this backup method, the TiDB cluster must be deployed on AWS EKS and uses AWS EBS volumes.
+- This backup method is currently not supported for TiFlash, TiCDC, DM, and TiDB Binlog.
+
+## Backup process
+
+You can either fully or incrementally back up snapshots based on AWS EBS volumes. The first backup of a node is a full, and subsequent backups are incremental. Snapshot backup is defined by a customized `Backup` custom resource (CR) object. TiDB Operator completes the backup task based on this object. If an error occurs during the backup process, TiDB Operator does not retry automatically. At this time, you need to handle it manually.
+
+The following sections exemplify how to back up data of the TiDB cluster `demo1`. `demo1` is deployed in the `test1` namespace in Kubernetes.
+
+### Step 1. Set up the environment for EBS volume snapshot backup
+
+1. Download the file [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml) to the backup server.
+
+2. Create the RBAC-related resources required for the backup in the `test1` namespace by running the following command:
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test1
+    ```
+
+3. Grant permissions to access remote storage.
+
+    To back up cluster metadata to Amazon S3, you need to grant the permissions to remote storage. Three ways are available. See [AWS account authorization](grant-permissions-to-remote-storage.md#aws-account-permissions).
+
+### Step 2. Back up data to S3 storage
+
+Based on the authorization method you selected in the previous step to grant remote storage access, you can back up data to S3 storage using any of the following methods accordingly:
+
++ Method 1: If you authorize permission by accessKey and secretKey, you can create the `Backup` CR as follows:
```

Comment:
```suggestion
+ Method 1: If you grant permissions by accessKey and secretKey, you can create the `Backup` CR as follows:
```

Reply:
reason: use the same verb as in https://docs.pingcap.com/tidb-in-kubernetes/stable/grant-permissions-to-remote-storage